R/last_fit.R

```diff
@@ -166,6 +175,32 @@ last_fit_workflow <- function(object,
     )
   }
 
+
+  if (inherits(split, "initial_validation_split")) {
```

Comment:
This block should go into a different function to make the main function more readable.

Reply:
adressed in [842a85a](https://github.com/tidymodels/tune/pull/701/commits/842a85a8241c24b84bdc5acc608bb30aac549aeb)